pkg/services/rpcsrv/server.go

```diff
@@ -1750,7 +1750,8 @@ func (s *Server) findStorageInternal(id int32, prefix []byte, start, take int, s
 	var (
 		i   int
 		end = start + take
-		res = new(result.FindStorage)
+		// Results is an empty list if a contract state is not found as it is in C# implementation.
```

Comment:
s/Results/res (if referring to a var) or s/Results is/Result is (if talking about generic operation result).